src/syncAzure.proj

```diff
@@ -10,6 +10,9 @@
     <BlobNamePrefix>$(__BlobNamePrefix)</BlobNamePrefix>
     <DownloadDirectory Condition="'$(PublishTestNativeBins)' != 'true'">$(PackagesDir)AzureTransfer</DownloadDirectory>
     <DownloadDirectory Condition="'$(PublishTestNativeBins)' == 'true'">$(PackagesDir)TestNativeBins\$(RuntimeId)</DownloadDirectory>
+
+    <!-- We don't want to run sync on test native binaries for Windows, since those don't get published -->
+    <SyncingNativeOnWindows Condition="'$(PublishTestNativeBins)' == 'true' and $(RuntimeId.ToLower().StartsWith('win'))">true</SyncingNativeOnWindows>
```

Comment:
It's the latter - I wanted it available at the time "Build" gets defined so that Build could be conditioned on it - I don't want it to be an error condition, since the point of this is to not exit with failure. Are you suggesting just changing the Condition for "Build" to use the two properties that define SyncingNativeOnWindows? I could get behind that